docs/changelog.rst

```diff
@@ -0,0 +1 @@
+../CHANGELOG.rst
```

Comment:
Is this correct? I thought a `.. include:` derivative is needed.

Reply:
Github doesn't show file flags, but that is a symlink.

Reply:
Oh, makes sense, thanks.